addons/fcs/functions/fnc_canUseFCS.sqf

```diff
@@ -18,3 +18,4 @@
 getNumber ([configFile >> "CfgVehicles" >> typeOf vehicle ACE_player, [ACE_player] call EFUNC(common,getTurretIndex)] call EFUNC(common,getTurretConfigPath) >> QGVAR(Enabled)) == 1
 && {cameraView == "GUNNER"}
 && {!([ACE_player] call CBA_fnc_canUseWeapon)} //Not Turned Out
+&& {cameraOn != (getConnectedUAV ACE_player)} //Not Controlling a UAV
```

Comment:
When zues RCing another unit `getConnectedUAV player` and `getConnectedUAV ace_player` are both null